### DISCUSSION TITLE
Problems with Home Assistant OS VM after upgrade (no internet on host but supervisor does)

### DISCUSSION THREAD

**Discussion by malosaa** [tteck/Proxmox#1746] [Help Needed]
Hi, 

i always used Home Assistant OS VM from tteck scripts, worked perfect.
since yesterday i updated hass os and my supervisor has internet but my host doesnt... 

i did open this topic, as i really don't know where to look at.
https://community.home-assistant.io/t/partial-no-internet-on-hass-proxmox-after-hass-os-update/609018/34
Is it an issue from proxmox ? or home assistant.

However i installed on a total new system (latest version of proxmox)
did run `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/vm/haos-vm.sh)"`
and it installed HASS OS, however it keeps hanging on (making it ready) because when i login shell and type (ha network info) 
host has no internet but supervisor does.

Any idea if this container has an issue ?

as all other containers do all work normal and have all internet.

regards

**Comment by tteck** [comment]
> even on my new system its stuck on the (getting ready page)

It's likely that the issue is related to your network configuration or settings.

**Reply by tteck** [reply]
But again, It worked before 🤷🏻‍♂️

**Reply by malosaa** [reply]
really weird, 

my network settings are ok.
but the reason i post this question here is because i'm out of options.
we tried everything, even some suggestions on the hass forum, or other reddit posts, none of them work at all.

is there a way to see somehow in that container why its blocked, as i can ping every ip from within that one and i can reach homeassistant from outside my network, only thing is that i cant update any updates.

when using the job command to ignore the local host, it updates, but thats a security issue.. and i get i direct warning that my system is not healthy.. so i really don't know where to look at :(

**Reply by malosaa** [reply]
just weird if i have an pending update with this command (ha jobs options --ignore-conditions internet_host) it updates normal but i cant retrieve new updates, so somewhere the container is making an issue i think..

**Reply by tteck** [reply]
Just out of curiosity, have you attempted to create a new virtual machine using the development branch to check if the problem still persists?

**Reply by malosaa** [reply]
yes i buyed a new pc today, installed proxmox and did run your hass OS script.
same issue, but i really don't know if its a proxmox issue, as i have no windows pc here to test.

**Reply by tteck** [reply]
Did you create the VM using the dev branch (Advanced settings) to check if the problem still persists?

**Reply by malosaa** [reply]
no sorry i did not, never done that.

i just borrowed a laptop and installed hassOS latest version on windows using virtualbox, getting host_internet = false.
so now we know its not proxmox at lease. 
so i gonna try an older version and see..

sorry to have put the question here, but i am out of options, i will see if there is some issue with my dns.
but on my pfsense i did even got rid of all dns options in there and put it back to stock see if that helped, but no..

**Reply by tteck** [reply]
No problem, I'd recommend giving the dev branch a shot. This way, you can verify whether the issue still remains upstream.

**Reply by malosaa** [reply]
thanks for the advice, i will try that 👍
